gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
Since these two pages are assembled server-side, I think the should be in the `dynamicUrlToDependencies` object so the service worker cache will get updated if  the templates change. I think that would look like this:

```
dynamicUrlToDependencies: {
  '/1.0/': [ '/1.0/index.html', '/templates/head-meta.html', '/templates/site-nav.html' ],
 '/1.0/about':  [ '/1.0/about.html', '/templates/head-meta.html', '/templates/site-nav.html' ]
}
```